frontend/src/metabase/dashboard/reducers.unit.spec.js

```diff
@@ -31,6 +31,7 @@ describe("dashboard reducers", () => {
         loadingStatus: "idle",
       },
       parameterValues: {},
+      draftParameterValues: {},
```

Comment:
Fixed default value on `draftParameterValues` so it looks the same as `parameterValues` to avoid confusion.